.github/workflows/build-and-test-backend.yml

```diff
@@ -17,19 +17,29 @@ env:
 jobs:
   build-and-test-backend:
     name: build-and-test-${{matrix.os}}
-    runs-on: ubuntu-latest
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
 
     steps:
-      - uses: actions/checkout@v2
-      - name: Setup .NET Core
-        uses: actions/setup-dotnet@v1
+      - uses: actions/checkout@v3
+
+      - name: Setup .NET Core ${{ env.DOTNET_VERSION }}
+        uses: actions/setup-dotnet@v3
         with:
           dotnet-version: ${{ env.DOTNET_VERSION }}
 
       - name: Install dependencies
         working-directory: ./backend
         run: dotnet restore
 
+      - name: Check licenses
+        working-directory: ./backend
+        run: |
+          dotnet tool install --global dotnet-project-licenses --version 2.7.1
+          dotnet-project-licenses -i src/api --allowed-license-types allowed-licenses.json
```

Comment:
Would it be possible to use a wildcard for a path to project instead of hardcoded value? For example if We add src/dataaccess it We will need to update this place as well. Can We make it like src/* or something similar?

Reply:
Good catch. I changed it to src so now it will scan all solutions defined there.